src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -352,9 +354,9 @@ private void AddExamples(MamlCommand command)
                     AddParagraphs(example.Introduction);
                 }
 
-                if (example.Code != null)
+                for (var i = 0; example.Code != null && i < example.Code.Length; i++)
                 {
-                    AddCodeSnippet(example.Code);
+                    AddCodeSnippet(example.Code[i].Text, example.Code[i].LanguageMoniker, true);
```

Comment:
1) when passing a constant (`true`) use named parameter for readability
2) Let's put it behind a switch `-GuessCodeSnippetsLanguage` or maybe something shorter...

Reply:
Done